### ISSUE TITLE
export project modal

### ISSUE THREAD

**Issue by maxbates** [Autodesk/genetic-constructor#942]
- file naming
- list blocks - want one or all

**Comment by maxbates** [comment]
@jlachoff - lets go over this?

think we are at the point where we want to configure export -- naming of files, whether you want one or all combinations, etc (or that might be it). Any thoughts on this initial modal, before we have real naming? Might just be a prefix for emily's use?

**Comment by cojofra** [comment]
no additional design
we do need to figure out which text fields we wanted - @jlachoff check the recoding

**Comment by jlachoff** [comment]
I checked the recording, but unfortunately we only recorded the portion of the session when she was demonstrating her APE workflow, and she did not talk about it there. In my written notes from the conversation, I captured, "Filenaming: EA0807 - EA1413" and "roundtripping to APE". It's possible that the numbering is useful to her for bringing these files BACK into GC after doing some processing elsewhere. I can reach out to her to confirm if it's okay with you.
